lib/msal-browser/src/interaction_client/PlatformAuthInteractionClient.ts

```diff
@@ -127,26 +127,32 @@ export class NativeInteractionClient extends BaseInteractionClient {
         );
 
         const extensionName =
-            this.nativeMessageHandler.getExtensionId() ===
+            this.platformAuthProvider.getExtensionId() ===
             NativeConstants.PREFERRED_EXTENSION_ID
                 ? "chrome"
-                : this.nativeMessageHandler.getExtensionId()?.length
+                : this.platformAuthProvider.getExtensionId() ===
+                  NativeConstants.MICROSOFT_ENTRA_BROKERID
+                ? NativeConstants.MICROSOFT_ENTRA_BROKERID
+                : this.platformAuthProvider.getExtensionId()?.length
                 ? "unknown"
                 : undefined;
```

Comment:
Instead of this complex ternary can we add a property to `platformAuthProvider` that indicates the name?